network/test/testUtil.go

```diff
@@ -32,53 +34,63 @@ var rootBlockID string
 
 const DryRun = true
 
-var allocator *portAllocator
-
 // init is a built-in golang function getting called first time this
 // initialize the allocated ports map and the root block ID
 func init() {
-	allocator = newPortAllocator()
 	rootBlockID = unittest.IdentifierFixture().String()
 }
```

Comment:
We can remove the `init` and just do `var rootBlockID string = unittest.IdentifierFixture().String()`

Reply:
fixed in https://github.com/onflow/flow-go/pull/195/commits/c186a2ab7051f4c03e6b50dfd16c028758c21c13